code/ui/generalhud/gametimer/GameTimer.cs

```diff
@@ -37,10 +41,29 @@ public override void Tick()
                     return;
                 }
 
+                lang = ILanguage.GetActiveLanguage();
```

Comment:
For now it's okay, but I think I gonna rework this a bit with own text labels and language change refreshes so we can handle them with a good performance :)

Reply:
What are your ideas with handling this? I think we should just have a single `language` object that exists either on the top-level hud or somewhere on the game instance, instead of getting the active language for each UI element.

In terms of handling the language change, it could be as simple as refreshing the hud.

Reply:
@Alf21
First I need to learn how to track cvar changes.